sonar-project.properties

```
@@ -0,0 +1,23 @@
+# Must be unique in a given SonarQube instance
+sonar.projectKey=opticks
+
+# This is the name and version displayed in the SonarQube UI.
+sonar.projectName=Opticks - Toggle Library
+sonar.projectVersion=1.0
+ 
+# Path is relative to the sonar-project.properties file.
+sonar.sources=src
+sonar.tests=src
+ 
+# Encoding of the source code. Default is default system encoding
+sonar.sourceEncoding=UTF-8
+
+sonar.exclusions=**/*.test.*
```

You might want to exclude mocks as well to correctly reflect the code coverage.

Maybe smth like
```suggestion
sonar.exclusions=**/*.test.*,src/mocks/*,
```